src/robusta/core/reporting/base.py

```diff
@@ -94,6 +95,9 @@ def __init__(
         finding_type: FindingType = FindingType.ISSUE,
         failure: bool = True,
         creation_date: str = None,
+        fingerprint: str = None,
+        starts_at: datetime = datetime.now(),
```

Comment:
There is a bug here. Default parameters in Python are evaluated only once, so `starts_at` will always be equal to the time when this function was first imported, not the time of the function call.

You can try this for yourself (see below):
<img width="566" alt="Screen Shot 2022-06-09 at 20 14 30" src="https://user-images.githubusercontent.com/494087/172905830-30019938-ef9d-44ba-82a4-edefafc33a2f.png">

Reply:
The common solution is to set `start_at=None` and in the function itself if it is None then you set to `datetime.now()`

Reply:
oops. Good catch!
Fixing